action.yml

```diff
@@ -1,5 +1,5 @@
-name: Versatile PyInstaller
-author: '@sayyid5416'
+name: PyInstaller-Nac
```

Comment:
Revert the **action and author name** to its original form, if you want me to merge it.